### ISSUE TITLE
Get-PartnerCustomerUsageSummary throws converting exception

### ISSUE THREAD

**Issue by boecskoer** [Microsoft/Partner-Center-PowerShell#24]
# Steps to reproduce

Just execute statement e.g.
`PS C:\> Get-PartnerCustomerUsageSummary -CustomerId $customer.CustomerId`
Module Version:
```
PS C:\> Get-InstalledModule -Name PartnerCenter | FT -AutoSize

Version     Name          Repository Description                                                              
-------     ----          ---------- -----------                                                              
1.0.1809.10 PartnerCenter PSGallery  Microsoft Partner Center - cmdlets for managing Partner Center resources.
```
## Expected behavior

should return Customer Usage Summary

## Actual behavior

```
PS C:\> Get-PartnerCustomerUsageSummary -CustomerId $customer.CustomerId
Get-PartnerCustomerUsageSummary : Object of type 'Microsoft.Store.PartnerCenter.Models.Usage.SpendingBudget' cannot be converted to type 
'Microsoft.Store.PartnerCenter.PowerShell.Models.Usage.PSSpendingBudget'.
At line:1 char:1
+ Get-PartnerCustomerUsageSummary -CustomerId $customer.CustomerId
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-PartnerCustomerUsageSummary], ArgumentException
    + FullyQualifiedErrorId : System.ArgumentException,Microsoft.Store.PartnerCenter.PowerShell.Commands.GetPartnerCustomerUsageSummary
```

## Environment

Windows 10 Enterprise 1709
PowerShell 5.1 Build 16299 Revision 611

**Comment by isaiahwilliams** [comment]
Hi @boecskoer, 

Thank you for bringing this issue to my attention. I have modified the code to correct this issue and to prevent it from happening again. The fix for this issue will be deployed later this week because I am still performing regression testing on a handful of other features that will be added at the same time. Please let us know if you find any other issues.

**Comment by isaiahwilliams** [comment]
Hi @boecskoer, 

I just published version [1.5.1810.2](https://www.powershellgallery.com/packages/PartnerCenter/1.5.1810.2) which includes a fix for this issue. Once you update to the latest version this command will function as excepted. Please let me know if you find any other issues.

**Comment by boecskoer** [comment]
Hi @isaiahwilliams thank you for fixing this issue in record time!
